lib/openhab/dsl/rules/automation_rule.rb

```diff
@@ -110,8 +110,10 @@ def create_queue(inputs)
         def extract_event(inputs)
           event = inputs&.dig("event")
           unless event
-            event = Struct.new(:event, :attachment, :command).new
-            event.command = inputs&.dig("command")
+            input_keys = inputs.to_h.keys.grep(/^[a-z_]+[a-z0-9_]*$/).grep_v("module") # only pick valid identifiers
```

Comment:
add `A-Z` to the second character class.

Reply:
I forgot to add case insensitivity flag